spec/requests/pokemon_spec.rb

```diff
@@ -6,28 +6,29 @@
       get "/pokemon"
       payload = JSON.parse(response.body)
       # puts "Paylod Empty = #{payload}"
-      # expect(payload).to be_empty
+      expect(payload).to be_empty
       expect(response).to have_http_status(:success)
     end
   end
 
   describe "GET /pokemon - With data in the DB" do
+    let!(:pokis) { create_list(:pokemon, 10) }
```

Comment:
If you don't refer to the variable in the test, you can also use a `before` block to create data before the tests run.